topo_processor/util/aws_credentials.py

```diff
@@ -1,6 +1,6 @@
 import json
 import os
-from typing import Dict
+from typing import Any, Dict
```

Comment:
Unused `Any` here?

Reply:
Fixed in https://github.com/linz/topo-processor/pull/512